xtask/src/xtask.rs

```diff
@@ -104,6 +104,24 @@ fn manpages(sh: &Shell) -> Result<()> {
         "cargo run --features=docgen -- man --directory target/man"
     )
     .run()?;
+
+    // Post-process hack to unconditionally define the roff string for
+    // apostrophe, See:
+    // https://github.com/bootc-dev/bootc/pull/1385#discussion_r2172661872
+    for page in std::fs::read_dir(sh.current_dir().join("target/man"))? {
+        let page = page?;
+        let path = page.path();
+        cmd!(sh, "sed -i")
+            .arg("-e")
+            .arg(r"1i .ds Aq \\(aq") // insert the groff variant unconditionally
+            .arg("-e")
+            .arg(r"/\.g \.ds Aq/d") // remove generated if conditional
+            .arg("-e")
+            .arg(r"/.el .ds Aq '/d") // remove generated else conditional
+            .arg(&path)
+            .run()?;
```

Comment:
I'm ok with this as but note a big part of the idea of `cmd!` is that it *properly shell quotes* automatically, so this would be more readable like

```
let groffsub = r"1i .ds Aq \\(aq";
let dropif = r"/\.g \.ds Aq/d";
let dropelse = r"/.el .ds Aq '/d";
cmd!(sh, "sed -i -e {groffsub} -e {dropif} -e {dropelse} {path}).run()?;
```

or so

Reply:
When I had it as part of the macro command itself and not chained off of the returned `Cmd`, it was giving some silly error about an unmatched `'` or something along those lines.  May have very well been something silly I was doing, though.  I'll double check it.

Reply:
:+1: ok it does work now like that, so obviously it was something I had wrong at the beginning.  Updated!